docs/identity-service.md

```diff
@@ -238,18 +238,22 @@ The returned `auth.keyHandle` value is meant to be used with the [Keys Service](
 
 ### Get IoT module identity information
 
-`GET /identities/modules/{module-id}?api-version=2020-09-01`
+`GET /identities/modules/{module-id}?api-version=2020-09-01[&type={type}]`
```

Comment:
I thought the final version of the API would have a required `type` parameter (not optional). So, should we reflect that now or later?

Reply:
I'll update it when we implement the change.